android/engine/src/main/java/org/smartregister/fhircore/engine/data/remote/model/response/UserResponse.kt

```diff
@@ -23,5 +23,10 @@ import kotlinx.serialization.Serializable
 data class UserInfo(
   @SerialName("questionnaire_publisher") val questionnairePublisher: String? = null,
   @SerialName("organization") val organization: String? = null,
-  @SerialName("location") val location: String? = null
+  @SerialName("location") val location: String? = null,
+  @SerialName("family_name") val familyName: String? = null,
+  @SerialName("given_name") val givenName: String? = null,
+  @SerialName("name") val name: String? = null,
+  @SerialName("preferred_username") val preferredUsername: String? = null,
+  @SerialName("sub") val sub: String? = null
```

Comment:
I am assuming that all this info is in custom user entity. so do we any property except publisher, location and sub?

Reply:
We get other properties in practitioner details response, using the same class to update user info once practitioner details are fetched.